.github/workflows/QA.yml

```diff
@@ -79,7 +79,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.10", "3.11", "3.12", "3.13"]
+        python-version: ["3.10", "3.11", "3.12", "3.13", "3.14"]
```

Comment:
watchdog 6.0.0 doesn’t (yet) declare/support Python 3.14, and its sdist for macOS bundles a prebuilt *_cpython-310-darwin.so, so when building on 3.14 the wheel ends up with mismatched tags

Reply:
https://pypi.org/project/watchdog/#files